support-lambdas/bigquery-acquisitions-publisher/package.json

```diff
@@ -23,6 +23,7 @@
 		"@aws-sdk/client-ssm": "^3.750.0",
 		"@aws-sdk/credential-provider-node": "^3.750.0",
 		"@google-cloud/bigquery": "^7.9.2",
+    "google-auth-library": "^9.0.0",
```

Comment:
Somehow we got away without having an explicit dependency for this before but now we need one.

Reply:
Is the whitespace off here? (tabs vs spaces maybe?)